build/yarn.lock

```diff
@@ -585,7 +585,7 @@ extend@~1.2.1:
 "extensions-modules@file:../extensions-modules":
   version "0.1.0"
   dependencies:
-    dataprotocol-client "file:../../../../Library/Caches/Yarn/v1/dataprotocol-client"
+    dataprotocol-client "file:../../../Library/Caches/Yarn/v1/dataprotocol-client"
```

Comment:
what's this change? it doesn't look related to the slick grid update

Reply:
This is a side affect of yarn. we will continue to have these changes until we move the dataprotocol client out and use either github reference of npm reference.

Reply:
I'm guessing they're pretty much benign since I was able to build with the wrong path (for me) and travis can build with the wrong path (for it)